### PR TITLE
Zhaofeng/sharenow readme

### PR DESCRIPTION
**Pull request by xzf0587**
1. add readme for share now
2. improve manifest and icon

### CODE REVIEW

File: share-now/README.md [hunk 1]
```diff
@@ -1,4 +1,119 @@
...
+    GO
+    CREATE TABLE [UserVoteEntity](
+	    [VoteID] [int] PRIMARY KEY IDENTITY,
+	    [PostID] [int] NOT NULL,
+	    [UserID] [uniqueidentifier] NOT NULL,
+    )
+    GO
+    ```
+1. Once deployment is complete, open Debug View (`Ctrl+Shift+D`) and select "Launch Remote (Edge)" or "Launch Remote (Chrome)" in dropdown list.
+1. Press "F5" to open a browser window and then select your package to view todo list sample app.
```
Comment (adashen) [inline share-now/README.md:55]: todo list?

File: share-now/README.md [hunk 2]
```diff
@@ -1,4 +1,119 @@
...
+    BACKEND_SQL_USER_NAME=
+    BACKEND_SQL_PASSWORD=
+
+    BOT_SQL_ENDPOINT=
+    BOT_SQL_DATABASE_NAME=
+    BOT_SQL_USER_NAME=
+    BOT_SQL_PASSWORD=
+    ```
+1. Open Debug View (`Ctrl+Shift+D`) and select "Debug (Edge)" or "Debug (Chrome)" in dropdown list.
+1. Press "F5" to open a browser window and then select your package to view todo list sample app.
```
Comment (adashen) [inline share-now/README.md:97]: todo list?

File: share-now/README.md (hunk 1 above)
Comment (xzf0587) [reply share-now/README.md]: It is fixed. Thx,

File: share-now/README.md (hunk 2 above)
Comment (xzf0587) [reply share-now/README.md]: It is fixed. Thx,

**Review by adashen — APPROVED** [review]

**Pull request merged**